Fix:
Undefined symbols for architecture x86_64:
  "_term_bgp_debug_as4", referenced from:
      _assegments_parse in libbgp.a(bgp_aspath.o)
      _aspath_reconcile_as4 in libbgp.a(bgp_aspath.o)
ld: symbol(s) not found for architecture x86_64
https://stackoverflow.com/questions/25050575/basic-use-of-c-archives-libraries-with-clang-linker
--- bgpd/bgp_debug.c.orig	2022-01-18 07:13:13.000000000 -0600
+++ bgpd/bgp_debug.c	2022-01-18 07:13:27.000000000 -0600
@@ -50,7 +50,7 @@
 unsigned long conf_bgp_debug_allow_martians;
 unsigned long conf_bgp_debug_nht;
 
-unsigned long term_bgp_debug_as4;
+unsigned long term_bgp_debug_as4 = 0;
 unsigned long term_bgp_debug_fsm;
 unsigned long term_bgp_debug_events;
 unsigned long term_bgp_debug_packet;
